react/src/base/inputs/SprkCheckbox/SprkCheckboxItem/SprkCheckboxItem.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import uniqueId from 'lodash/uniqueId';
+
+const SprkCheckboxItem = (props) => {
+  const {
+    children,
+    variant,
+    idString,
+    additionalClasses,
+    analyticsString,
+    checkboxAdditionalClasses,
+    labelAdditionalClasses,
+    name,
+    value,
+    disabled,
```

Comment:
Talked with RV, this should be isDisabled to match our naming conventions. He's gonna make a PR for it.